src/providers/google/api.ts

```diff
@@ -0,0 +1,14 @@
+export interface GoogleFetchPayload {
+    apiKey: string
+    body: Record<string, any>
+}
+
+export const fetchChatCompletion = async(payload: GoogleFetchPayload) => {
+    const { apiKey, body } = payload || {}
+    const initOptions = {
+        headers: { 'Content-Type': 'application/json' },
+        method: 'POST',
+        body: JSON.stringify({ ...body }),
+    }
+    return fetch(`https://generativelanguage.googleapis.com/v1beta3/models/text-bison-001:generateText?key=${apiKey}`, initOptions);
```

Comment:
I think you might want to use `models/gemini-pro` instead of `models/text-bison-001`?

See https://ai.google.dev/models and curl:

```
curl https://generativelanguage.googleapis.com/v1/models/gemini-pro:generateContent?key=$API_KEY \
    -H 'Content-Type: application/json' \
    -X POST \
    -d '{ "contents":[
      { "parts":[{"text": "Write a story about a magic backpack"}]}
    ]
}'
```

Reply:
My apologies. This URL points to PaLM2 instead of Gemini Pro, I'll fix it ASAP

Reply:
Apologies for my mistake of using PaLM2 as the Gemini API. 
I take those words back:
- ~Google Gemini currently does not support Chinese.~ It does support Chinese. [API Doc](https://ai.google.dev/available_regions#available_languages). 
- Also Gemini API has decent dialog capabilities.

## Description

- Remove the PaLM2 model because it've become a legacy. Now users can select `gemini-pro` for a chat.
- Modified the process of using the Gemini API conversation.

## Additional context

- It looks like the number of lists passed in must be odd, not even, for multi-round conversations. [Another Repo Issue](https://github.com/songquanpeng/one-api/issues/836). The corresponding code is in the [src/logics/conversation.ts](https://github.com/anse-app/anse/pull/115/commits/c2897584465614d39d119a1813993e4eb08d5477#diff-a8949d5d681f905b6972e20c6f3e5bb37a36b35e72702c7d858e4b0295307bf5R47)
- The Gemini api also supports passing in parameters such as `temperature`, `topk`, etc.. I will add them in the future.